components/layout/BenefitListItem.tsx

```diff
@@ -0,0 +1,69 @@
+// @description - see example in get-involved benefits of joining us section
+import {
+  Avatar, Box, ListItem, ListItemAvatar, ListItemText,
+} from '@mui/material';
+import { styled } from '@mui/system';
+import React from 'react';
+
+interface BenefitItemProps {
+  id: number;
+  title: string;
+  leftOffset: number;
+  rightOffset: number;
+  isSmallScreen?: boolean;
+}
+
+interface BenefitBoxProps {
+  leftoffset: number; // custom DOM elements must be all lowercase
+  rightoffset: number;
+}
+
+const BenefitBox = styled(Box)<BenefitBoxProps>(({ leftoffset, rightoffset, theme }) => ({
+  paddingLeft: `${leftoffset}rem`,
+  paddingRight: `${rightoffset}rem`,
+  paddingBottom: theme.spacing(3),
+}));
+
+const BenefitListItem = styled(ListItem)(({ theme }) => ({
+  backgroundColor: theme.palette.primary.main,
+  width: '90%',
+  height: '100%',
+  paddingTop: 0,
+  paddingBottom: 0,
+  background: 'linear-gradient(to right, #546B95 70%, rgba(217, 217, 217, 0) 100%)',
+  borderRadius: 15.50,
+}));
+
+const ItemNumber = styled(Avatar)(() => ({
+  fontSize: '1.4rem',
+  width: '1.875rem',
+  height: '1.875rem',
+  backgroundColor: 'transparent',
+  color: 'white',
+  border: '0.125rem solid white',
+}));
+
+export default function BenefitItem({
+  id, title, leftOffset, rightOffset, isSmallScreen = false,
+}: BenefitItemProps) {
+  let adjustedLeftOffset = leftOffset;
+  let adjustedRightOffset = rightOffset;
+
+  if (isSmallScreen) {
+    adjustedLeftOffset = 0;
+    adjustedRightOffset = 0;
+  }
```

Comment:
```suggestion
  const theme = useTheme();
  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
  const adjustedLeftOffset = isSmallScreen ? 0 : leftOffset;
  const adjustedRightOffset = isSmallScreen ? 0 : rightOffset;
```
I dont think its necessary to receive as prop this `isSmallScreen`.
We can just use inside component and let him manage this using the hook for that.

Pass too many props to a component can lead us to a code smell, so would be good to just pass really necessary props.